x-pack/test/security_solution_cypress/cypress/screens/inspect.ts

```diff
@@ -286,12 +284,12 @@ export const INSPECT_BUTTONS_IN_SECURITY: InspectButtonMetadata[] = [
         tab: EVENTS_TAB,
         id: EVENT_CONTAINER_TABLE_NOT_LOADING,
       },
-      {
-        title: 'User risk',
-        tab: RISK_SCORE_TAB,
-        id: RISK_SCORE_TAB_CONTENT,
-        customIndexPattern: 'ml_user_risk_score_latest_default',
-      },
+      // {
```

Comment:
@machadoum is there a way for us to reliably enable risk score for the associated test?